tests/snapd

```diff
@@ -99,12 +90,19 @@ done
 if [ "${ID}" != "ubuntu-core" ]; then
     (
         set -e
-        usermod -G lxd -a jenkins || true
+
+        test_user="testuser"
+        useradd -m ${test_user} || true
+        usermod -G lxd -a ${test_user}
         LXC=$(which lxc)
-        su jenkins -c "${LXC} info"
+        su ${test_user} -c "${LXC} info"
+        userdel -r ${test_user} 2>/dev/null || true
     )
 fi
 
+# Check that preseed via snap works
+lxc config show | grep -qF "core.https_address: '[::]:8443'" 
```

Comment:
Ah, I now understand the earlier `snap set lxd daemon.preseed`. Would you mind putting the comment+check closer to the `snap set`?